src/Sulu/Bundle/CustomUrlBundle/Request/CustomUrlRequestProcessor.php

```diff
@@ -163,11 +163,9 @@ private function matchCustomUrl($url, PortalInformation $portalInformation, Requ
      * Server encodes the url and symfony does not encode it
      * Symfony decodes this data here https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/Routing/Matcher/UrlMatcher.php#L91.
      *
-     * @param $pathInfo
-     *
      * @return string
      */
-    private function decodeUrl($pathInfo)
+    private function decodeUrl(string $pathInfo)
```

Comment:
we cannot change the method signature here too

Reply:
How should we go forward with this topic?

Reply:
remove the typehint and add it as phpdoc 🙂